src/app/shared/doc-viewer/docs-api-module-import/docs-api-module-import.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, Input, AfterViewInit, OnInit } from "@angular/core";
+import { MatSnackBar } from "@angular/material/snack-bar";
+import { CopierService } from "../../copier/copier.service";
+
+const EXPORT_NAME_PLACEHOLDER = "{{exportName}}";
+const MODULE_IMPORT_PATH_PLACEHOLDER = "{{moduleImportPath}}";
+const MODULE_IMPORT_TEMPLATE = `import ${EXPORT_NAME_PLACEHOLDER} from '${MODULE_IMPORT_PATH_PLACEHOLDER}';`;
+
+@Component({
+  selector: "docs-api-module-import",
```

Comment:
Based on PR https://github.com/angular/components/pull/19200, should this be `'[docs-api-module-import]'`?